### ISSUE TITLE
[QUESTION] or [BUG] starting retopoflow, mesh turns strange

### ISSUE THREAD

**Issue by nowhereeverywhere** [CGCookie/retopoflow#1260]
Retopoflow Version: {*3.4*}

Blender Version/Hash: {*3.6*}

Platform Version/Distribution: {*window11*}

**Issue:**

I've started retopoflow, but it didn't work well.
I'm sorry i couldn't elaborate. I leave screenshots instead.
I really hope my problem is solved. help me plz

{*add any screenshots of the issue here*}

![스크린샷 2023-09-04 132553](https://github.com/CGCookie/retopoflow/assets/143986437/ef1e97c6-a656-47d8-8d1d-bd0f16c6c175)
this is before retopoflow

![스크린샷 2023-09-04 132632](https://github.com/CGCookie/retopoflow/assets/143986437/f2180df9-c01c-4091-b4fe-0045094f8118)
This is what happens when i run retopoflow

**Comment by vxlcoder** [comment]
thanks for the report, @nowhereeverywhere .

any chance you can send us your blend file? retopoflow@cgcookie.com

there is something different happening with your file that we haven't seen yet.

thanks!

**Comment by nowhereeverywhere** [comment]
> thanks for the report, @nowhereeverywhere .
> 
> any chance you can send us your blend file? [retopoflow@cgcookie.com](mailto:retopoflow@cgcookie.com)
> 
> there is something different happening with your file that we haven't seen yet.
> 
> thanks!

I sent you my blender file. Please check it and tell me any problem.
THANKS!

**Comment by vxlcoder** [comment]
@nowhereeverywhere, thanks for sharing your .blend file!

i am able to reproduce this consistently on my machine, but it seems to only happen once... after starting RF, exit RF and then jump into edit mode on the source seems to make the issue go away, even jumping back into RF, but i'm not sure why yet.

Everything looks fine before starting RF.

![before starting RF](https://github.com/CGCookie/retopoflow/assets/3790699/7cc9fdcb-b335-48c4-8b4e-d75315a3cb19)

But after starting RF, the visual issues show.  note: this is only visual, as none of the tools appear to be affected by the isssues.

![after starting RF](https://github.com/CGCookie/retopoflow/assets/3790699/32c93a67-d398-44c3-bfae-9ed9548e9f59)

![RF tools still work as expected](https://github.com/CGCookie/retopoflow/assets/3790699/6ecc8821-7c92-40fc-92dd-1f209052596f)

When exiting RF, the visual artifacts are still there.  this implies that the issue is a Blender-related problem (we don't render the source meshes, only the target mesh)

![after quitting RF](https://github.com/CGCookie/retopoflow/assets/3790699/c5249c34-b9fd-4d05-b292-5131af66d69d)

Editing the source mesh _after_ starting RF sort of "resets" the issue.

![editing source mesh](https://github.com/CGCookie/retopoflow/assets/3790699/06fe6442-589f-4d11-8223-fad595108ef4)

Even when jumping into RF again.

![started RF again](https://github.com/CGCookie/retopoflow/assets/3790699/9e0bdbec-0279-4e7c-8240-44b062e0cb7f)

![using polystrips](https://github.com/CGCookie/retopoflow/assets/3790699/72f40b83-fbcb-4b2b-a4ec-e00179a7b4c1)

A note: jumping into edit mode _before_ starting RF does _not_ prevent the issue.  it's only a workaround after RF has been started already.  but, adding a decimate modifier before starting RF with a ratio < 1.0 (0.99, 0.75, etc.) on the `human` source _does_ prevent the visual issue.

Did you append this mesh from a different .blend file or import from an external file?

i'm wondering if there are some invalid properties in the mesh (coordinates and normals seem fine, so just in topology?) that are causing a problem.

**Comment by nowhereeverywhere** [comment]
@vxlcoder **I'm really grateful for your thorough help.** i couldn't have done it without you. I thoroughly read your comment.
Following your advice, I added a decimate modifier before starting RF, and it was successful!

This is what I sculpted without using an external file.
so I'm not sure what's causing the issue.

**Comment by vxlcoder** [comment]
what external program did you use? what format did you use (obj, ply, etc)?

you might be able to apply the modifier to have it permanently fix the visual bug, but I didn't try that.

**Comment by nowhereeverywhere** [comment]
@vxlcoder 
Oh, I sculpted **without** using an external file.
I didn’t import other file; just sculpted mesh!

Or Is there something I'm missing? I’m not fluent in English or blender programs.

**Comment by chrischameleon** [comment]
I actually had this same problem just a few hours ago, I came here to see if anyone had a fix and @vxlcoder your fix worked of tab into edit mode and tab out.
I sculpted my mesh in Blender and added a decimate mod to it, Retopoflow worked fine as long I didn't apply the decimate but when I applied it would explode the mesh as soon as I entered into retopoflow.

**Comment by nowhereeverywhere** [comment]
oh. In my case, it worked well without applying it, 
and even if I exit to apply decimate and operate it again, the mesh stays well.

**Comment by jlampel** [comment]
This definitely looks like a Blender bug. They fixed a vertex index order bug in recent 4.0 builds that looked very similar but it doesn't change this file's results. It could be worth reporting if it we find out what part of the RF startup triggers it. Just scaling by a matrix doesn't seem to be it.

**Comment by jfranmatheu** [comment]
Closing issue due to prolonged inactivity. In the past discussion is believed to be a past Blender mesh-related issue that should be currently fixed, with no clear clues on RF implications. If the issue persist in latest Blender and RF versions (as well as a new .blend created in latest version), please feel free to open a new issue, thanks!
